Azure - FAQ doc updated (#4123)
SumoLogic/sumologic-documentation: docs/send-data/collect-from-other-data-sources/azure-monitoring/arm-integration-faq.md

### How do I export Azure function logs?

> All the Azure Function are enabled with application insights. To export Azure function logs, do the following:

Reviewer: ```suggestion
All Azure functions are enabled with application insights. To export Azure function logs, do the following:
```

> 1. Navigate to your **Azure Function** app from **App Catalog**.

Reviewer: ```suggestion
1. Go to the Sumo Logic **App Catalog** and navigate to your **Azure Function** app.
```

**Total = 47.5$**

### How do I troubleshoot an Event Hub Integration?

> For detailed information, see the [Azure Event Hubs Source for Logs Troubleshooting section](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/#troubleshooting). This includes testing the function using [**Sample** Logs](https://s3.amazonaws.com/appdev-cloudformation-templates/TestPayload.json) & [Metrics](https://s3.amazonaws.com/appdev-cloudformation-templates/metrics_fixtures.json).

Reviewer 2: ```suggestion
For detailed information, see the Azure Event Hubs Source for Logs [Troubleshooting](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/#troubleshooting) section. This includes testing the function using [**Sample** Logs](https://s3.amazonaws.com/appdev-cloudformation-templates/TestPayload.json) and [Metrics](https://s3.amazonaws.com/appdev-cloudformation-templates/metrics_fixtures.json).
```

> For detailed information, see the [Troubleshooting log collection](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/#troubleshooting) section of the Collect logs for Azure Monitor page. This includes testing the function using **Sample** [Logs](https://s3.amazonaws.com/appdev-cloudformation-templates/TestPayload.json) & [Metrics](https://s3.amazonaws.com/appdev-cloudformation-templates/metrics_fixtures.json).

Reviewer: ```suggestion
For detailed information, see the [Azure Event Hubs Source for Logs Troubleshooting section](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/#troubleshooting). This includes testing the function using [**Sample** Logs](https://s3.amazonaws.com/appdev-cloudformation-templates/TestPayload.json) & [Metrics](https://s3.amazonaws.com/appdev-cloudformation-templates/metrics_fixtures.json).
```
